content/en/docs/concepts/storage/storage-classes.md

```diff
@@ -791,8 +791,8 @@ kubectl create secret generic storageos-secret \
 --namespace=default
 ```
 
-Secrets used for dynamically provisioned volumes may be created in any namespace
-and referenced with the `adminSecretNamespace` parameter. Secrets used by
+The Secrets used for dynamically provisioned volumes may be created in any namespace
+and referenced with the `adminSecretNamespace` parameter. The Secrets used by
```

Comment:
“The Secrets used by pre-provisioned volumes must be created in the same namespace as the PVC that references it.” doesn't (to me) read like idiomatic English. You could either:
- keep the original sentence
- rewrite this sentence

*or* make a case that adding “The” helps readers enough to make it worth the slightly stilted language.

(which approach do you prefer?)

Reply:
@sftim , yeah, I feel good to retain the earlier approach itself. So reverted those changes. 
Thanks for the detailed comments.